index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Your indentation in this file is all kinds of out of whack.  I think it's a mashup between the autocomplete for HTML using tabs and you using spaces?